dev/provider_packages/prepare_provider_packages.py

```diff
@@ -1676,10 +1676,12 @@ def verify_changelog_exists(package: str) -> str:
 def list_providers_packages():
     """List all provider packages."""
     providers = get_all_providers()
-    # For now we should exclude open-lineage from being consider for releasing until it is ready to
-    # be released
-    if "openlineage" in providers:
-        providers.remove("openlineage")
+    # if provider needs to be not considered in release add it here
+    # this is useful for cases where provider is WIP for a long period thus we don't want to release it yet.
+    providers_to_remove_from_release = []
+    for provider in providers_to_remove_from_release:
+        if provider in providers:
+            providers.remove(provider)
     for provider in providers:
         console.print(provider)
```

Comment:
Why do the in-place modification? We can just stick a `if provider in providers_to_remove_from_release: continue` in the latter for loop instead.

Reply:
It's not the same list.
We have list of all providers and we have providers to exclude. I just converted the providers to exclude into a list.

Reply:
My question was why you need to modify the list at all. I was suggesting

```python
for provider in providers:
    if provider in providers_to_remove_from_release:
        continue
    console.print(provider)
```